girder/constants.py

```diff
@@ -195,6 +195,12 @@ class TokenScope:
     ANONYMOUS_SESSION = 'core.anonymous_session'
     USER_AUTH = 'core.user_auth'
     TEMPORARY_USER_AUTH = 'core.user_auth.temporary'
+    PLUGINS_ENABLED_READ = 'core.plugins.read'
```

Comment:
Of course, that capability depends on a feature that doesn't yet exist: applications should be able to do a lookup to the server in which they send a list of all the API calls they wish to make and are sent back a set of required scope identifiers. If the apps do this lookup prior to sending the user through the login flow, they'll never lose compatibility.